src/Files.App/BaseLayout.cs

```diff
@@ -760,15 +714,16 @@ private void AddShellItemsToMenu(List<ContextMenuFlyoutItemViewModel> shellMenuI
             {
                 var openWith = contextMenuFlyout.SecondaryCommands.FirstOrDefault(x => x is AppBarButton abb && (abb.Tag as string) == "OpenWith") as AppBarButton;
                 var flyout = openWithOverflow.Flyout as MenuFlyout;
-                flyout.Items.Clear();
+
+                flyout!.Items.Clear();
```

Comment:
I'm not convinced we should be using "!" so much just to get rid of some nullable warnings. E.g. this one could have been possibly better solved by:
```cs
var flyout = (MenuFlyout)openWithOverflow.Flyout;
flyout.Items.Clear();
```

Reply:
Ok, I'll change that